src/components/Select.js

```diff
@@ -38,6 +41,11 @@ const Select = React.createClass({
     };
   },
 
+  componentDidMount () {
+    styles.option[':hover'].backgroundColor = this.props.color || StyleConstants.Colors.PRIMARY;
+    styles.activeItem.backgroundColor = this.props.color || StyleConstants.Colors.PRIMARY;
```

Comment:
Actually, rather than do that. We've implemented a new pattern that hasn't made it's way to our open source components. Instead of defining our styles in a `const`, we now define them in a `styles` method. We then reference that method when needing styles. This gives us easier access to props and state.

```javascript
getDefaultProps () {
  return {
    color: StyleConstants.Colors.PRIMARY
  },
},

render () {
  const styles = this.styles();

  // other render code
},

styles () {
  return {
    // other styles
    activeItem: {
      backgroundColor: this.props.color
    },
    option: {
      // other styles
      ':hover': {
        backgroundColor: this.props.color
      }
    }
  };
}